test/e2e/examples.go

```diff
@@ -84,13 +90,13 @@ var _ = KubeDescribe("[Feature:Example]", func() {
 
 			By("checking up the services")
 			checkAllLogs := func() {
-				forEachPod(c, ns, "name", "redis", func(pod api.Pod) {
+				forEachPod("name", "redis", func(pod api.Pod) {
```

Comment:
why was c & ns trim'd out of the signature.  Seems like they are still leveraged in the function are now relying on scope.